x-pack/plugins/siem/public/components/stat_items/index.tsx

```diff
@@ -177,13 +207,13 @@ export const StatItemsComponent = React.memo<StatItemsProps>(
           <EuiFlexGroup>
             {enableBarChart && (
               <FlexItem>
-                <BarChart barChart={barChart!} />
+                <BarChart barChart={barChart!} configs={barchartConfigs} />
               </FlexItem>
             )}
 
             {enableAreaChart && (
               <FlexItem>
-                <AreaChart areaChart={areaChart!} />
+                <AreaChart areaChart={areaChart!} configs={areachartConfigs} />
```

Comment:
Side note, above this file you have a misspelling:

```ts
isBarChartDataAbailable
```

should be:

```ts
isBarChartDataAvailable
```

Reply:
Thanks for pointing this out!